### ISSUE TITLE
Holding Alt+Shift to go in reverse from the first time.

### ISSUE THREAD

**Issue by pushqrdx** [sigoden/window-switcher#63]
The way `alt+tab` works right now is that I have to first click alt tab which goes n + 1 and then if hold shift, I can go in reverse, but this means I have to `alt+shit+tab` twice, it'd be nice if `alt+shit+tab` initiated the switcher in reverse mode from the first time.

**Comment by pushqrdx** [comment]
@sigoden fwiw my proposed change is basically how the built-in alt tab behaves, if you initiate it holding shift it will start going in the opposite direction.

**Comment by pushqrdx** [comment]
@sigoden for some strange reason #64 brought back the issue in #62

**Comment by sigoden** [comment]
@pushqrdx I never solved #62.

**Comment by sigoden** [comment]
@pushqrdx Everything is fine, I see no problems

**Comment by pushqrdx** [comment]
@sigoden Hmmm, I am not actually sure what happened on my end, I reverted back to #61 version that was working and for some reason it's not working anymore either.

**Comment by pushqrdx** [comment]
@sigoden this does not happen for some strange reason, i minimize C, B, A in that order, focus another application, press `alt+tab`, notepad DOES NOT appear at the end of the list, I select notepad, C.txt gains focus.

**Comment by pushqrdx** [comment]
@sigoden this does not happen anymore. I minimize C, B, A in that order, focus another application, press `alt+tab`, notepad DOES NOT appear at the end of the list, I select notepad, C.txt gains focus.

For some very strange reason, #62 stopped happening and I actually get what I expected in #62 where last minimized application appears next to currently focused one. which is very bizarre as it wasn't doing that earlier when I reported #62.

Here's a video:

https://github.com/sigoden/window-switcher/assets/25884937/98a57a60-4249-457e-99ad-203943bfa8e6

**Comment by sigoden** [comment]
Are you running application build with last commit ? I was not able to reproduce the problem in your video.

**Comment by pushqrdx** [comment]
@sigoden yes latest commit, but I also tried to revert to #61 but it had the exact same behavior.

**Comment by sigoden** [comment]
https://github.com/sigoden/window-switcher/assets/4012553/1dd240f3-1309-4ce5-b4f9-b3475d31eaab

@pushqrdx see, everything is ok. I don't understand why it doesn't work on your computer

After all notepad windows are minimized in order, run the following code to check the window order  is correct?

```
cargo run -p inspect-windows | grep Notepad
```

Is the result like the belowing?
```
visible:✓cloacked iconic✓topmost:  8193700:C.txt - Notepad 0:
visible:✓cloacked iconic✓topmost:  8456964:B.txt - Notepad 0:
visible:✓cloacked iconic✓topmost:  1638874:A.txt - Notepad 0:
visible: cloacked iconic topmost:  6161950:Default IME 8193700:C.txt - Notepad
visible: cloacked iconic topmost:  3672784:Default IME 8456964:B.txt - Notepad
visible: cloacked iconic topmost:  8194546:Default IME 1638874:A.txt - Notepad
```

**Comment by pushqrdx** [comment]
@sigoden I don't understand either. #62 is no longer happening which is strange, I am kinda glad it works but this is magic. I didn't do anything. 
In your video notepad moves to the end of the stack when the last window is minimized but you can see in my video this doesn't happen anymore and instead it's correctly placed after current active app
